vipps-ecom-api.md

```diff
@@ -851,6 +852,18 @@ Response:
 }
 ```
 
+### Cancelling a pending order
+
+If you wish to cancel a transaction before the customer can finish you can send a cancel request while the transaction is in the "Initate" stage. This is intended for situations where you for example have a face to face situation where a customers phone runs out of battery. This should not be considered a consistent guranteed operation as the Cancel request by you the Merchant is independent by any actions taken by the user in the app.
+
+This will lead to the transaction being
+
+``` 
+status: "Cancelled"
```

Comment:
Response in `/cancel` call is `$.transactionInfo.status: "Cancelled"` but /details is `$.transactionLogHistory.0.operation: "CANCEL"`
Yay for consistency

Reply:
💀💀💀💀